mmv1/third_party/terraform/services/iap/resource_iap_settings_test.go

```diff
@@ -0,0 +1,107 @@
+package iap_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccIapSettings_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckIapSettingsDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
Can you also add the update test? Doc is here https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test

Reply:
Sure